src/core/mempool.rs

```diff
@@ -714,7 +723,8 @@ impl MemPoolDB {
     ///  highest-fee-first order.  This method is interruptable -- in the `settings` struct, the
     ///  caller may choose how long to spend iterating before this method stops.
     ///
-    ///  Returns the number of transactions considered on success.
+    ///  `todo` must return a result of type StacksTransactionResult, indicating the outcome of the
+    ///  transaction.
```

Comment:
These comment changes don't seem accurate, right? `todo` still returns a `bool` indicating whether or not to continue iterating or exit early.